src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -47,6 +47,11 @@ export default class CenterPanel extends React.PureComponent<Props, State> {
     this.setState({ isLoading: true })
   }
 
+  componentWillUnmount() {
+    const { onSetWearablePreviewController } = this.props
+    onSetWearablePreviewController(null)
```

Comment:
Setting the wearable preview controller as null will work, but it won't close the channel because the unsubscribe function won't be ever executed.
It might be harmless but the correct way to do it is to emit an END message in the controller when the wearable preview component gets unmounted.

Reply:
thanks for the feedback! working on it